### PR TITLE
Organization command

### PR DESCRIPTION
**Pull request by arturgontijo**
Related: #42

### CODE REVIEW

File: snet_cli/arguments.py [hunk 1]
```diff
@@ -325,6 +333,187 @@ def add_service_options(parser, config):
...
+    _add_organization_arguments(p)
+
+    org_create_p = subparsers.add_parser("create", help="Create an Organization", default_choice="default")
+    org_create_p.set_defaults(fn="create")
+    networks_organization_subparsers = org_create_p.add_subparsers(title="networks", metavar="[NETWORK]")
+    org_create_p.add_argument("name", help="Name of the Organization", metavar="ORG_NAME")
+    org_create_p.add_argument("members", help="List of members to be added to the organization",
+                              metavar="ORG_MEMBERS[]")
+
+    for network_name in network_names:
```
Comment (vsbogd) [inline snet_cli/arguments.py]: As far as I understand for each organization command the list of networks is added as additional options for 
the command. On the other hand root parser already has --network option to specify the network. Can it be used here?

**Comment by vsbogd** [comment]
It is worth to update https://github.com/singnet/snet-cli/blob/master/README.md to add new commands.

File: snet_cli/arguments.py (hunk 1 above)
Comment (arturgontijo) [reply snet_cli/arguments.py]: Yes, I think we can get `network` from `config` (session).
I'll change it and make some tests to see if it's OK.

**Comment by vsbogd** [comment]
Looks good for me, have finished the tests you mentioned at https://github.com/singnet/snet-cli/pull/50#discussion_r218894835 ?

**Comment by arturgontijo** [comment]
Yes but only at `argparse` level.
I didn't use [eth-teste](https://github.com/ethereum/eth-tester) yet.

**Pull request merged**